### DISCUSSION TITLE
No Icons are visible

### DISCUSSION THREAD

**Discussion by MeditatedBison** [end-4/dots-hyprland#412] [Q&A] [answered]
Really grateful to you for providing these dotfiles. 
Icons are not visible and it is looking very weird. this is the problem i am facing.

i am using Arch linux with Hyprland. just installed fresh yesterday
When i ran the script . there was one error about that there are two conflicting dependencies. 
One was hyprland-git-0.38... i think, and the other one was 0.39.000, so i ignored the error. 

But after complete installation and reboots. I have a system , in which icons of things are not visible, as you can see in the photo
thanks in advance 
![20240414_20h31m52s_grim](https://github.com/end-4/dots-hyprland/assets/95769851/e818f73d-c6c8-46fa-bd84-89f0048c0c35)

**Comment by end-4** [comment]
> as you can see in the photo

ya forgot to upload it

**Reply by MeditatedBison** [reply]
In the top bar of the uploaded photo, The wifi logo, bluetooth logo is not visible, And text is written everywhere, How can i fix that?

Thanks

**Reply by MeditatedBison** [reply]
![20240414_20h43m49s_grim](https://github.com/end-4/dots-hyprland/assets/95769851/c1f84f67-2656-4466-bfcd-f686d71224da)

**Reply by end-4** [reply, marked as answer]
okay it does show for me now. whoops
you wanna install the Material Symbols font (`ttf-material-symbols-variable-git` aur package)
run `fc-cache -fv` or reboot afterwards to refresh font cache

**Reply by MeditatedBison** [reply]
Thank you so much i installed the Material Symbols font, And did a reboot and it worked!!

**Reply by ghost** [reply]
awesome, was looking for this fix :)

**Comment by fatsheep2** [comment]
I encountered the same problem, and installing the ttf-material-symbols-variable-git package did not solve my issue. The result of executing the fc-cache -fv command is as follows:

fc-cache -fv
Font directories:
/home/fatsheep/.local/share/fonts
/usr/local/share/fonts
/usr/share/fonts
/home/fatsheep/.fonts
/usr/share/fonts/OTF
/usr/share/fonts/TTF
/usr/share/fonts/adobe-source-code-pro
/usr/share/fonts/adobe-source-han-serif
/usr/share/fonts/cantarell
/usr/share/fonts/encodings
/usr/share/fonts/liberation
/usr/share/fonts/noto
/usr/share/fonts/noto-cjk
/usr/share/fonts/sarasa-gothic
/usr/share/fonts/ttf-gabarito-git
/usr/share/fonts/ttf-material-symbols-variable
/usr/share/fonts/ttf-readex-pro
/usr/share/fonts/twemoji
/usr/share/fonts/wenquanyi
/usr/share/fonts/encodings/large
/usr/share/fonts/wenquanyi/wqy-zenhei
/home/fatsheep/.local/share/fonts: caching, new cache contents: 0 fonts, 0 dirs
/usr/local/share/fonts: skipping, no such directory
/usr/share/fonts: caching, new cache contents: 0 fonts, 15 dirs
/usr/share/fonts/OTF: caching, new cache contents: 3 fonts, 0 dirs
/usr/share/fonts/TTF: caching, new cache contents: 663 fonts, 0 dirs
/usr/share/fonts/adobe-source-code-pro: caching, new cache contents: 30 fonts, 0 dirs
/usr/share/fonts/adobe-source-han-serif: caching, new cache contents: 7 fonts, 0 dirs
/usr/share/fonts/cantarell: caching, new cache contents: 6 fonts, 0 dirs
/usr/share/fonts/encodings: caching, new cache contents: 0 fonts, 1 dirs
/usr/share/fonts/encodings/large: caching, new cache contents: 0 fonts, 0 dirs
/usr/share/fonts/liberation: caching, new cache contents: 12 fonts, 0 dirs
/usr/share/fonts/noto: caching, new cache contents: 2137 fonts, 0 dirs
/usr/share/fonts/noto-cjk: caching, new cache contents: 80 fonts, 0 dirs
/usr/share/fonts/sarasa-gothic: caching, new cache contents: 480 fonts, 0 dirs
/usr/share/fonts/ttf-gabarito-git: caching, new cache contents: 6 fonts, 0 dirs
/usr/share/fonts/ttf-material-symbols-variable: caching, new cache contents: 0 fonts, 0 dirs
/usr/share/fonts/ttf-readex-pro: caching, new cache contents: 6 fonts, 0 dirs
/usr/share/fonts/twemoji: caching, new cache contents: 1 fonts, 0 dirs
/usr/share/fonts/wenquanyi: caching, new cache contents: 0 fonts, 1 dirs
/usr/share/fonts/wenquanyi/wqy-zenhei: caching, new cache contents: 3 fonts, 0 dirs
/home/fatsheep/.fonts: skipping, no such directory
/usr/share/fonts/OTF: skipping, looped directory detected
/usr/share/fonts/TTF: skipping, looped directory detected
/usr/share/fonts/adobe-source-code-pro: skipping, looped directory detected
/usr/share/fonts/adobe-source-han-serif: skipping, looped directory detected
/usr/share/fonts/cantarell: skipping, looped directory detected
/usr/share/fonts/encodings: skipping, looped directory detected
/usr/share/fonts/liberation: skipping, looped directory detected
/usr/share/fonts/noto: skipping, looped directory detected
/usr/share/fonts/noto-cjk: skipping, looped directory detected
/usr/share/fonts/sarasa-gothic: skipping, looped directory detected
/usr/share/fonts/ttf-gabarito-git: skipping, looped directory detected
/usr/share/fonts/ttf-material-symbols-variable: skipping, looped directory detected
/usr/share/fonts/ttf-readex-pro: skipping, looped directory detected
/usr/share/fonts/twemoji: skipping, looped directory detected
/usr/share/fonts/wenquanyi: skipping, looped directory detected
/usr/share/fonts/encodings/large: skipping, looped directory detected
/usr/share/fonts/wenquanyi/wqy-zenhei: skipping, looped directory detected
/var/cache/fontconfig: not cleaning unwritable cache directory
/home/fatsheep/.cache/fontconfig: cleaning cache directory
/home/fatsheep/.fontconfig: not cleaning non-existent cache directory
fc-cache: succeeded

My system has undergone a reinstallation, but user data has been retained. I suspect it might be due to caching. If so, how should I clear the cache to reinstall the fonts? I've looked for many solutions but none have worked. Please help me, thank you very much~
